netsync/manager.go

```diff
@@ -79,6 +79,13 @@ type headersMsg struct {
 	peer    *peerpkg.Peer
 }
 
+// notFoundMsg packages a Decred notfound message and the peer it came from
```

Comment:
It isn't really a decred message at this point so the comment is a bit misleading.

Reply:
Thanks. Fixed.